uktrade/data-workspace-frontend
# Rich text editor - Hide heading level options

### Description of change
When we had the previous rich text editor(CKEditor) we where hiding some of the heading level options such as H1,H2,H5 and H6. As we are now using the [TinyMCE](https://github.com/uktrade/data-workspace-frontend/pull/3238) editor we should fully suit. Currently the user can can choose these heading levels however if they do, we still remove the HTML around the content so this is just becomes confusing for them. This change just uses CSS to hide those options.

![Screenshot 2024-08-02 at 13 32 49](https://github.com/user-attachments/assets/969b482a-2a2e-424d-a6b7-b1fc15036fa1)

### How to test:

For full test instructions refer to #3238 to find all the rich text editors.

1. Go to any rich text editor on the front end and you should NOT be able to change selected text to a H1,H2,H5 or H6
2. Go to any rich text editor on the backend end `/admin` and you should NOT be able to change selected text to a H1,H2,H5 or H6

### Checklist
